File: guanabara-java/src/com/danillo/algoritmos/lista09/Exercicio91.java
package com.danillo.algoritmos.lista09;

import java.util.Scanner;

public class Exercicio91 {
    private static void maior(int a, int b) {
        if (a > b){
            System.out.println("Maior valor: " +a);
        } else if (b > a){
            System.out.println("Maior valor: " +b);
        } else{
            System.out.println("Os valores são iguais.");
        }
    }

    public static void main(String[] args){
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira um número: ");
        int a = entrada.nextInt();

        System.out.println("Insira outro número: ");
        int b = entrada.nextInt();

        maior(a, b);
    }
}

File: guanabara-python/src/lista09/exercicio91.py
a = int(input("Digite um número inteiro: "))
b = int(input("Digite outro número inteiro: "))


def maior(a, b):
    if a > b:
        print("Maior valor: ", a)
    elif b > a:
        print("Maior valor: ", b)
    else:
        print("Os valores são iguais.")


maior(a, b)
